app/presenters/tree_builder_alert_profile_obj.rb

```diff
@@ -0,0 +1,60 @@
+class TreeBuilderAlertProfileObj < TreeBuilder
+  def initialize(name, type, sandbox, build = true, assign_to = nil, cat = nil, objects = nil)
+    @assign_to = assign_to
+    @cat = cat
+    @objects = objects
+    @cat_tree = true if @assign_to.ends_with?("-tags")
+    build = false unless @assign_to
+    super(name, type, sandbox, build)
+  end
+
+  def override(node, object, _pid, _options)
+    identifier = (object.name.presence || object.description)
+    identifier += "-" + object.hostname if object.kind_of?(MiddlewareServer)
+    node[:title] = identifier
+    node[:icon] = "fa fa-tag" if @cat_tree
```

Comment:
app/decorators/classification_decorator.rb has fonticon method. Please rewrite it so it returns pficon pficon-folder-close for Classification that has entries and otherwise fa fa-tag. Something like entries.present? ? "pficon pficon-folder-close" : "fa fa-tag" should do :)